src/interlocking_component/route_controller.py

```diff
@@ -55,7 +61,7 @@ def insert_platform_unblocked(self, platform: Platform):
         :param platform: the unblocked platform
         :type platform: Platform
         """
-        raise NotImplementedError()
+        self.route_controller.recalculate_all_routes()
 
     def insert_track_speed_limit_changed(self, track: Track):
```

Comment:
I am wondering, weather these two methods should (for now) just be deleted. Speed of trains or the tracks is not (yet) taken into account.

Reply:
Yes, that sounds even better